scripts/maskfilter.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from scipy.ndimage import binary_erosion, binary_dilation, generate_binary_structure
+from skimage.measure import label, regionprops
+import numpy as np
+from util import load_nifti, save_nifti
+import sys
+from os.path import abspath
+
+struct_element= generate_binary_structure(3,1)
+
+def findLargestConnectMask(img):
+
+    mask = label(img > 0, connectivity=1)
+    maxArea = 0
+    for region in regionprops(mask):
+        if region.area > maxArea:
+            maxLabel = region.label
+            maxArea = region.area
+
+    largeConnectMask = (mask == maxLabel)
+
+    return largeConnectMask
+
+
+
+def single_scale(InputImage, ss):
+
+    # erosion
+    morph_image= binary_erosion(InputImage, struct_element, iterations= ss)
```

Comment:
This maskfilter is simpler than mrtrix maskfilter which is implemented in https://github.com/pnlbwh/CNN-Diffusion-MRIBrain-Segmentation/blob/master/src/maskfilter

It is a simplified version aimed at smoothing the boundary edges.